pkg/subgraph/src/mappings/cv-strategy.ts

```diff
@@ -278,3 +279,69 @@ export function handlePowerDecreased(event: PowerDecreased): void {
 
   member.save();
 }
+
+export function handleDecayUpdated(event: DecayUpdated): void {
+  let cvs = CVStrategy.load(event.address.toHexString());
+  if (cvs == null) {
+    log.debug("handleDecayUpdated cvs not found: {}", [
+      event.address.toHexString(),
+    ]);
+    return;
+  }
+  if (cvs.config) {
+    let config = CVStrategyConfig.load(cvs.config);
+    if (config == null) {
+      log.debug("handleDecayUpdated config not found: {}", [
+        event.address.toHexString(),
+      ]);
+      return;
+    }
+    config.decay = event.params.decay;
+    config.save();
+  }
+  return;
```

Comment:
that its assuming that always cvs config need be created before, never assume that because if we change the contracts and remove defaults maybe that dont happen, so check if not exist and create if its necessary.

Reply:
We can not create the config entity from one of this events since it has a bunch of mandatory properties that we have not here